src/components/Header.module.css

```diff
@@ -1,15 +1,61 @@
 .header {
-  display: flex;
-  justify-content: space-around;
+  background-color: #fff;
+  border: 1px solid #e8e8e8;
   align-items: center;
+  justify-content: space-between;
+  padding: 1em 3em 1.2em 3em;
+  gap: 1em;
+  display: flex;
+  flex-direction: row;
 }
 
-.navigation ul {
+.nav {
+  padding: 24px 4%;
+  background-color: #fff;
   display: flex;
-  list-style: none;
-  gap: 1em;
+  flex-direction: row;
+  align-items: center;
+  gap: 239px;
 }
 
-.navigation ul a {
+.nav ul a {
   text-decoration: none;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  opacity: 1;
+  color: #6e6e6b;
+}
+
+.nav ul {
+  display: flex;
+  position: fixed;
+  list-style-type: none;
+  gap: 30px;
+  margin-left: -51%;
+}
+
+nav h1 {
+  display: flex;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.875rem;
+  font-weight: 700;
+  color: #0290ff;
+  gap: 10px;
+}
+
+.im_icon {
+  border: 1px solid #d3d3d3;
+  color: #0290ff;
+  padding: 1em;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
```

Comment:
- [ ] You have finished most of the design on this part, but please update the icon on the width and height as on the design because currently it looks way bigger:
- [ ] Also the categories navbar header needs to have a different color, kindly update it


Yours:
![image](https://github.com/aima98/Bookstore/assets/62705947/a37165f4-6cdf-4baa-ad30-4e78c19c8503)

Design:
![image](https://github.com/aima98/Bookstore/assets/62705947/21410aba-4175-41c5-841f-e3af7fa3dede)